etc/frontend/di.xml

```diff
@@ -1,9 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:ObjectManager/etc/config.xsd">
+<config
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:ObjectManager/etc/config.xsd">
     <type name="Magento\Checkout\Model\CompositeConfigProvider">
         <arguments>
             <argument name="configProviders" xsi:type="array">
-                <item name="omise_config_provider" xsi:type="object">Omise\Payment\Model\Ui\CcConfigProvider</item>
+                <item name="cc_config_provider" xsi:type="object">Omise\Payment\Model\Ui\CcConfigProvider</item>
```

Comment:
@jacstn wonder if it will be any duplicated-name issue.. would it be more safe to have `omise_` as prefix?